src/main/java/com/seailz/discordjar/model/channel/GuildChannel.java

```diff
@@ -111,4 +118,86 @@ default void addPermissionOverwrite(@NotNull PermissionOverwrite overwrite) {
     default CreateChannelInviteAction createInvite() {
         return new CreateChannelInviteAction(discordJv(), id());
     }
+
+    /**
+     * Creates an Incoming Webhook for this channel.
+     *
+     * @param name The name of the Webhook.
+     * @return The created Webhook.
+     * @implNote Avatars are not yet implemented in discord.jar.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    // FIXME: 3/23/23 Implement avatar data
+    default IncomingWebhook createWebhook(String name) throws DiscordRequest.UnhandledDiscordAPIErrorException {
```

Comment:
Put this in the impl class of GuildChannel
```suggestion
   IncomingWebhook createWebhook(String name) throws DiscordRequest.UnhandledDiscordAPIErrorException;
```